src/components/LinePlotContainer/LinePlotContainer.tsx

```diff
@@ -1,73 +1,33 @@
 import React from 'react'
-import { Grid, CardContent, Card, ButtonGroup, Button, Paper } from '@material-ui/core'
-
+import { Grid, CardContent, Card, Button, Paper } from '@material-ui/core'
 import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos'
 import LinePlot from '@components/LinePlot/LinePlot'
-import { data as mockData } from './mockData'
-import { colors } from '@static/theme'
-
 import useStyles from './style'
 interface Data {
   id: string
-  data: Array<{ x: number; y: number }>
+  points: Array<{ x: number; y: number }>
 }
-
-export const LinePlotContainer: React.FC = () => {
+interface IProp {
+  data: Data[]
+}
+export const LinePlotContainer: React.FC<IProp> = ({ data }) => {
   const classes = useStyles()
-  const [menuOption, setMenuOption] = React.useState('Volument')
-  const [activeButtonTime, setActiveButtonTime] = React.useState('Y')
-  const [data, setData] = React.useState<Data>(mockData[0])
-  const [dataPlot, setDataPlot] = React.useState<Array<{ x: number; y: number }>>(
-    mockData[0].data.sort((a: { x: number; y: number }, b: { x: number; y: number }) => {
-      return a.x - b.x
-    })
-  )
+  const [menuOption, setMenuOption] = React.useState('Volume')
+  const [dataTmp, setDataTmp] = React.useState<Data>(data[0])
   const changeData = (name: string) => {
-    const value = mockData.findIndex(element => element.id === name)
-
-    setData(mockData[value])
-  }
-
-  const sortData = (type: string) => {
-    const timestamp = Date.now()
-    if (type === 'D') {
-      setDataPlot(
-        data.data.filter(element => {
-          return element.x > timestamp - 86400000
-        })
-      )
-    }
-    if (type === 'W') {
-      setDataPlot(
-        data.data.filter(element => {
-          return element.x > timestamp - 604800000
-        })
-      )
-    }
-    if (type === 'M') {
-      setDataPlot(
-        data.data.filter(element => {
-          return element.x > timestamp - 2629743000
-        })
-      )
-    }
-    if (type === 'Y') {
-      setDataPlot(
-        data.data.filter(element => {
-          return element.x > timestamp - 31556926000
-        })
-      )
-    }
+    const value = data.findIndex(element => element.id === name)
+    setDataTmp(data[value])
   }
-
   React.useEffect(() => {
-    sortData(activeButtonTime)
-  }, [activeButtonTime, data.data])
+    changeData(menuOption.toLowerCase())
```

Comment:
there might be problem when menuOption is set to "User count", because in data there is "userCount", not "user count"